### PR TITLE
Set up @interactors/material-ui in new repo

### PR DESCRIPTION
**Pull request by cherewaty**
## Motivation
Combine `interactors` repos.

## Approach
- Pull contents and git history of `material-ui-interactors` into this repo.
- Clean up lint tool dependencies.

## Open question
The `material-ui-interactors` repo was using `covector`, not `changesets`. Does `changesets` work with the `.alpha.minor` scheme in use here?

## After merge
- [ ] Move issues from https://github.com/thefrontside/material-ui-interactors
- [ ] Archive https://github.com/thefrontside/material-ui-interactors
- [ ] `npm deprecate` `material-ui-interactors`

### CODE REVIEW

**Comment by frontsidejack** [comment]
The preview packages of this pull request have been published.
Click on the following packages for instructions on how to install them:
<details><summary>@interactors/html</summary>
  
  ---
  Install using the following command:
  
  ```bash
  $ npm install @interactors/html@material-ui-move
  ```
  Or update your package.json file:
  
  ```bash
  {
    "@interactors/html": "material-ui-move"
  }
  ```
  
  ---
  </details><details><summary>@interactors/material-ui</summary>
  
  ---
  Install using the following command:
  
  ```bash
  $ npm install @interactors/material-ui@material-ui-move
  ```
  Or update your package.json file:
  
  ```bash
  {
    "@interactors/material-ui": "material-ui-move"
  }
  ```
  
  ---
  </details><details><summary>@interactors/with-cypress</summary>
  
  ---
  Install using the following command:
  
  ```bash
  $ npm install @interactors/with-cypress@material-ui-move
  ```
  Or update your package.json file:
  
  ```bash
  {
    "@interactors/with-cypress": "material-ui-move"
  }
  ```
  
  ---
  </details>
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against eee44a76519cdef77e28b2553cff625842ab932e
</p>

**Comment by cowboyd** [comment]
> The material-ui-interactors repo was using covector, not changesets. Does changesets work with the .alpha.minor scheme in use here?

@jbolda any insight here?

File: packages/material-ui/.prettierignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.test.tsx
```
Comment (cowboyd) [inline packages/material-ui/.prettierignore:1]: We're not using prettier, so we shouldn't need a .prettierignore

**Review by cowboyd — APPROVED** [review]

File: packages/material-ui/.prettierignore (hunk 1 above)
Comment (wKich) [reply packages/material-ui/.prettierignore:1]: I use it. There is no way to ignore some files from VSCode prettier extension settings. We can place that file on top level

**Review by wKich — APPROVED** [review]

**Pull request merged**